docs/gds-supported-platform/resizing-persistent-volumes.md

```diff
@@ -0,0 +1,93 @@
+# Resizing persistent volumes
+
+The volumes you created in the beginning may not be big enough for
+your ongoing needs.
+
+As an example, we're going to double the space on the Concourse Worker
+persistent volumes - from 64GiB to 128GiB. Replace
+`gsp-concourse-worker` with the name of the resource you want to
+resize the volume for.
+
+1. Make a change to `charts/gsp-system/values.yml` with your desired
+   sizing, like [in PR 343](https://github.com/alphagov/gsp/pull/343):
+
+```yaml
+...
+persistence:
+  worker:
+    size: 128Gi
+```
+
+1. When merged, the PR will be released to sandbox. Changes must pass
+   through sandbox via the full release process to make sure we haven't
+   broken anything before going to the Verify cluster.
+
+1. Pods in StatefulSets each have a volume. For Kubernetes to be able
+   to resize the volumes, we need to detach them from the pods, by
+   destroying the StatefulSet. To destroy the StatefulSet, run:
+
+   ```sh
+   $ gds sandbox kubectl -n gsp-system delete statefulset --cascade=false gsp-concourse-worker
+   ```
+
+   You should destroy the StatefulSet when the Concourse release pipeline starts to fail. You will see an error message similar to:
+
+    > The StatefulSet “gsp-concourse-worker” is invalid: spec:
+    > Forbidden: updates to statefulset spec for fields other than
+    > ‘replicas’, ‘template’, and ‘updateStrategy’ are forbidden
+
+   The `--cascade=false` flag makes sure that you do not delete the pods inside the StatefulSet.
+
+   The `-deployer` pipeline will re-apply the StatefulSet. This will
+   intentionally roll all the associated pods.
+
+1. Once the pods are in a `Running` state, edit the
+   `PersisentVolumeClaim`s manually.
+
+    - Find the name of the claims you want to delete. In this example,
+      we want the three `concourse-worker`s.
+
+      ```sh
+      $ gds sandbox kubectl -n gsp-system get persistentvolumeclaim
+      NAME                                        STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+      concourse-work-dir-gsp-concourse-worker-0   Bound    pvc-a633fb0a-76fe-11e9-bbf3-0ae9bf7a1ac0   64Gi      RWO            gp2            84d
+      concourse-work-dir-gsp-concourse-worker-1   Bound    pvc-a6366add-76fe-11e9-bbf3-0ae9bf7a1ac0   64Gi      RWO            gp2            84d
+      concourse-work-dir-gsp-concourse-worker-2   Bound    pvc-a6386683-76fe-11e9-bbf3-0ae9bf7a1ac0   64Gi      RWO            gp2            84d
```

Comment:
above, the text says `Pods in StatefulSets each have a volume`, but here it seems like all pods share the same volume?

Reply:
The volume IDs are different...

Reply:
oops, yes they are! 🙈